README.md

```diff
@@ -265,6 +265,14 @@ Specify the following `oauth.*` properties:
 Some authorization servers don't provide the `iss` claim. In that case you would not set `oauth.valid.issuer.uri`, and you would explicitly turn off issuer checking by setting the following option to `false`:
 - `oauth.check.issuer` (e.g. "false")
 
+You can enforce audience check, which is an OAuth2 mechanism to limit access to tokens that are explicitly issued for use by your protected server.
+The authorization server adds the allowed resource servers' `client ids` into `aud` claim of such tokens.
+Set the following option to `true` to enforce audience check (it is not enabled by default). 
+- `oauth.check.audience` (e.g. "true")
```

Comment:
So, the JWT token contains the `aud` field, right? And the `oauth.check.audience` says that it should be checked. But checked against what? Where do you set the allowed `aud` values?

Reply:
The valid value is only one - the configured `oauth.client.id`. Maybe it's not adequately explained in the README.md?
For example - if client id is configured as 'kafka' which it usually is, then the examples of valid `aud` value are:

* "kafka"
* ["rest-api", "kafka"]